### PR TITLE
Addition of FirelensConfiguration to the container

### PR DESCRIPTION
**Pull request by stack72**
Fixes: #490

This is defined in https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_ContainerDefinition.html

### CODE REVIEW

File: nodejs/awsx/ecs/container.ts [hunk 1]
```diff
@@ -270,6 +270,30 @@ export interface ContainerDependency {
...
+    options?: pulumi.Input<FirelensConfigurationOptions>;
+
+    /**
+     * The log router to use.
+     */
+    type: pulumi.Input<"fluentd" | "fluentbit">;
+}
+
+export interface FirelensConfigurationOptions {
+    enableEcsLogMetaData?: pulumi.Input<true | false>;
```
Comment (CyrusNajmabadi) [inline nodejs/awsx/ecs/container.ts]: is this not `Input<boolean>`?

Comment (stack72) [reply nodejs/awsx/ecs/container.ts]: d'oh! Complete brain fail :) Fixed - thanks for picking that up!

File: nodejs/awsx/ecs/container.ts [hunk 2]
```diff
@@ -270,6 +270,30 @@ export interface ContainerDependency {
...
+     */
+    options?: pulumi.Input<FirelensConfigurationOptions>;
+
+    /**
+     * The log router to use.
+     */
+    type: pulumi.Input<"fluentd" | "fluentbit">;
+}
+
+export interface FirelensConfigurationOptions {
```
Comment (lukehoban) [inline nodejs/awsx/ecs/container.ts]: I don't think this is right.  `options` is a "string to string map", not a strongly typed object.  The typing of this in https://github.com/pulumi/pulumi-aws/blob/d1b5c8a4765ae03efddb580e2d18f45d63d00a29/sdk/nodejs/ecs/container.ts#L73 is "correct" I believe.

Comment (stack72) [reply nodejs/awsx/ecs/container.ts]: good catch here! Thanks for that - updated now

**Pull request merged**